Holovibes/sources/compute/rendering.cc

```diff
@@ -242,42 +269,52 @@ void Rendering::insert_compute_autocontrast(std::atomic<bool>& autocontrast_requ
         if (!time_transformation_env_.gpu_time_transformation_queue->is_full())
             return;
 
-        if (autocontrast_request &&
-            (!image_acc_env_.gpu_accumulation_xy_queue || image_acc_env_.gpu_accumulation_xy_queue->is_full()))
+        if (apply_contrast(autocontrast_xy_, image_acc_env_.gpu_accumulation_xy_queue))
         {
             // FIXME Handle composite size, adapt width and height (frames_res =
             // buffers_.gpu_postprocess_frame_size)
```

Comment:
Maybe you could remove this fixme , it looks like an old comment

Reply:
Removed